docs/proposals/PROPOSAL_H.md

```diff
@@ -0,0 +1,271 @@
+# Proposal H - compatibility artifact in flat format
+
+The purpose of the compatibility artifact is to describe compatibility requirements for container images to run on a host.
+
+This proposal is based on proposal B, F and G with new ideas, some description is kept unchanged and borrowed from these original proposals.
+
+The design goal is to avoid over-flexible speficication, and avoid abusing the specification.
+
+It is NOT TO store variation of user configuration of applications in the compatibility artifact.
+
+The first part describes the compatibility artifact manifest.
+
+The second part focus on how the compatibility requirements to be described in flat format, including image selection and validation.
```

Comment:
```suggestion
The second part focuses on how the compatibility requirements are to be described in a flat format, including image selection and validation.
```
Including metadata attributes _for_ image selection and validation?

Reply:
ok, will update in next commit